valohai/internals/download.py

```diff
@@ -1,8 +1,39 @@
 import os
 import tempfile
+from requests import Response
+from valohai.internals.utils import uri_to_filename, check_sha256_hashing
 
 
 # TODO: This is close to valohai-local-run. Possibility to merge.
+def resolve_datum(datum_id: str) -> Response:
+    try:
+        from valohai_cli.api import request  # type: ignore
+    except ImportError as ie:
+        raise RuntimeError("Can't resolve datum without valohai-cli") from ie
+    resp: Response = request(url=f"/api/v0/data/{datum_id}", method="GET", stream=True)
```

Comment:
No need to stream the bit of JSON.
```suggestion
    resp: Response = request(url=f"/api/v0/data/{datum_id}", method="GET")
```